presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -333,6 +336,26 @@ private boolean checkTablePermission(ConnectorSecurityContext context, SchemaTab
         return false;
     }
 
+    private boolean checkTablePermissionAny(ConnectorSecurityContext context, SchemaTableName tableName, TablePrivilege... anyPrivileges)
+    {
+        if (INFORMATION_SCHEMA_NAME.equals(tableName.getSchemaName())) {
+            return true;
+        }
+
+        for (TableAccessControlRule rule : tableRules) {
+            Optional<Set<TablePrivilege>> tablePrivileges = rule.match(context.getIdentity().getUser(), tableName);
+            if (tablePrivileges.isPresent()) {
+                for (TablePrivilege anyPrivilege : anyPrivileges) {
```

Comment:
Use `TablePrivilege.values()` instead of `anyPrivileges`. This method is used in one place and so there is no need to for this method parameter.